test/functionalTest/cases/947_v2_entities/v2_entities_simple.test

```diff
@@ -0,0 +1,426 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Directory names are per issue number. Take into accoun that several issues may refer to the same operatoin (from a URL path point of view). In fact, the following issues number share the same /v2/entities path: #947, #952, #953 and (not yet filled) the query entities operation and the tests corresponding to each one should be in their separated directory.